website/client/src/components/groups/rightSidebar.vue

```diff
@@ -0,0 +1,231 @@
+<template>
+  <div class="sidebar px-4">
+    <div>
+      <div class="buttons-wrapper">
+        <div class="button-container button-with-menu-row">
+          <button
+            v-if="!isMember"
+            class="btn btn-success btn-success"
+            @click="$emit('join')"
+          >
+            <span v-once>{{ $t(isParty ? 'joinParty' : 'joinGuild') }}</span>
+          </button>
+          <button
+            v-if="isMember"
+            class="btn btn-primary inline"
+            @click="$emit('showInviteModal')"
+          >
+            <span v-once>{{ $t(isParty ? 'inviteToParty' : 'inviteToGuild') }}</span>
+          </button>
+          <b-dropdown
+            right="right"
+            toggle-class="with-icon"
+            class="ml-2"
+            :no-caret="true"
+          >
+            <template v-slot:button-content>
+              <span
+                class="svg-icon inline menuIcon"
+                v-html="icons.menuIcon"
+              >
+              </span>
+            </template>
+            <b-dropdown-item
+              v-if="isLeader && !group.purchased.active && group.privacy === 'private'"
+              class="selectListItem"
+              @click="$emit('upgradeGroup')"
+              :style="{'--hover-color': '#1ca372', '--hover-background': '#77f4c73F'}"
```

Comment:
why `:style` and hardcoded `#color` strings?

Reply:
need to revisit that - the hover color seems to be useable as one of the $scss vars but the hovher background with the special alpha channel might still end up as not a $scss var ... but lemme see when I get to the part

Reply:
the colors were changed - the hover backgrounds doesn't have any specific scss var